lib/decidim/comparative_stats/test/factories.rb

```diff
@@ -7,6 +7,6 @@
     endpoint { Faker::Internet.url }
     name { Faker::Name.name }
     version { Decidim::ComparativeStats::MIN_API_VERSION }
-    organization { create(:organization) }
+    organization
```

Comment:
why remove create organization?

Reply:
Since i create it while creating the endpoint i thought it was not necessary for it to be here. That way i was also using it for the users